docs/appguide.rst

```diff
@@ -351,7 +351,7 @@ to the watch:
     If the new code on the watch throws an exception (including an
     out-of-memory exception) then your watch will display a black screen at
     startup. If that happens, and you don't know how to debug the problem, then
-    you can use wasptool to restore the original version of main.py .
+    you can use wasptool to restore the original version of main.py.
```

Comment:
Do not remove the space after main.py. The "word" at the end of the sentance contains a period, so we add an extra space to delimit it unambiguously .